source/common/upstream/upstream_impl.cc

```diff
@@ -720,7 +720,8 @@ ClusterImplBase::partitionHostList(const HostVector& hosts) {
     }
   }
 
-  return {healthy_list, degraded_list, excluded_list};
+  return std::tuple<HealthyHostVectorConstSharedPtr, DegradedHostVectorConstSharedPtr,
```

Comment:
use `std::make_tuple`? ditto below

Reply:
Sure, I wanted to keep it as close to the original as possible, but happy to switch it out for `std::make_tuple`.